### PR TITLE
Invoke

### PR DESCRIPTION
**Pull request by trumae**

### CODE REVIEW

File: runtime/runtime_test.go [hunk 1]
```diff
@@ -217,58 +221,89 @@ func TestGoFunction(t *testing.T) {
...
+	}
+
+	_, err = c.TryEval(mustParse("(let respbytes (ioutil/ReadAll (. resp Body)))"))
+	if err != nil {
+		t.Fatalf("(ioutil/ReadAll (. resp Body)) should have generated an error.", err)
+	}
+
+	c.SetFn("bytes/NewBuffer", bytes.NewBuffer, CheckArity(1))
+	_, err = c.TryEval(mustParse("(let buf (bytes/NewBuffer respbytes))"))
+	if err != nil {
```
Comment (felipeweb) [inline runtime/runtime_test.go]: error message need always to be the same?

File: runtime/runtime_test.go [hunk 2]
```diff
@@ -28,6 +28,15 @@ func mustEval(s string) parser.Value {
 	return NewContext(nil).MustEval(mustParse(s))
 }
 
+func runSExpressions(c *Context, exprs []string, t *testing.T) {
```
Comment (avelino) [inline runtime/runtime_test.go]: Write godoc, I know it's private, it's important for future maintenance

File: runtime/runtime_test.go [hunk 3]
```diff
@@ -211,64 +225,54 @@ func TestGoFunction(t *testing.T) {
...
+	c.SetFn("ioutil/ReadAll", ioutil.ReadAll, CheckArity(1))
+	c.SetFn("bytes/NewBuffer", bytes.NewBuffer, CheckArity(1))
+
+	exprs := []string{
+		"(let resp (http/Get \"http://www.google.com/robots.txt\"))",
+		"(. resp Status)",
+		"(let respbytes (ioutil/ReadAll (. resp Body)))",
+		"(let buf (bytes/NewBuffer respbytes))",
+		"(. buf String)",
+		//"(print (. buf String))",
```
Comment (avelino) [inline runtime/runtime_test.go]: Why comment?

File: runtime/runtime.go [hunk 4]
```diff
@@ -476,3 +477,47 @@ func For(ctx *Context, args ...parser.Value) parser.Value {
 	}
 	return parser.NewAny(nil, nil)
 }
+
+//Dump the context content
+func (c *Context) Dump() {
+	for id, val := range c.env {
+		fmt.Println(id, val)
```
Comment (avelino) [inline runtime/runtime.go]: print?

Comment (trumae) [reply runtime/runtime.go]: I am using that func as debug tool. I accept suggestions.

Comment (felipeweb) [reply runtime/runtime.go]: maybe something in rumtime or rumtime/debug can help with this

Comment (avelino) [reply runtime/runtime.go]: We need to start thinking about package, the dump would enter the `debug` (#95) package.
more issue: #96

**Pull request merged**